components/ParameterForm.vue

```diff
@@ -118,27 +126,36 @@ const renderAsRadios = (parameter: Parameter) => {
   return parameter.parameterType === ParameterType.Select && optionsAreTerse(parameter);
 };
 
-const submitForm = () => {
-  // Not implemented yet
-};
+const formSubmitting = ref(false);
 
-const largeScreen = ref(true);
-const breakpoint = 992; // CoreUI's "lg" breakpoint
-const setFieldSizes = () => {
-  if (window.innerWidth < breakpoint) {
-    largeScreen.value = false;
-  } else {
-    largeScreen.value = true;
-  }
+const submitForm = async () => {
+  if (!formData.value) {
+    return;
+  };
+
+  formSubmitting.value = true;
+  const response = await $fetch<NewScenarioData>("/api/scenarios", {
+    method: "POST",
+    query: { // Using query instead of body because I couldn't work out how to send a body in the integration test.
+      parameters: formData.value,
+    },
+  }).catch((error: FetchError) => {
+    console.error(error);
```

Comment:
display this error back to user

Reply:
Making a ticket

Reply:
Context - I only expect there to be errors here in development, since there is no known way for the form data to fail validation at the moment (you can't empty the input fields)

Reply:
sweet